src/test/DeltaNeutralLp.t.sol

```diff
@@ -37,21 +43,29 @@ contract L1DeltaNeutralTest is TestPlus {
         vault = Vault(address(deployL1Vault()));
     }
 
-    function _usdc() internal virtual returns (address) {
-        return 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
+    function _asset() internal pure virtual returns (address) {
+        return 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48; //usdc
     }
 
     function _deployStrategy() internal virtual {
         strategy = Sslp.deployEth(vault);
     }
 
+    function _deployStrategyWithCustomParams(LendingParams memory params)
```

Comment:
The virtual functions were to make one base contract work with different pools on different networks. There's nothing network specific here, so you can delete this function and just deploy the strategy in `testCustomLendingParamsFuzz`